README.md

```diff
@@ -257,6 +256,8 @@ int wdog_kick2       (int id, unsigned int *ack);
 int wdog_extend_kick (int id, unsigned int timeout, unsigned int *ack);
 ```
 
+See [wdog.h](src/wdog.h) for detailed API documentation.
```

Comment:
I considered adding the full Doxygen comments here, but I think it might be too verbose for the README file.

Reply:
Yup, agreed.